src/ManagerZKit.ts

```diff
@@ -0,0 +1,127 @@
+import fs from "fs";
+import https from "https";
+import os from "os";
+import path from "path";
+import process from "process";
+import { v4 as uuid } from "uuid";
+
+import { ManagerZKitConfig, ManagerZKitPrivateConfig, PtauInfo, TemplateType } from "./types";
+
+export class ManagerZKit {
+  private _config: ManagerZKitPrivateConfig;
+
+  constructor(config: Partial<ManagerZKitConfig>) {
+    const projectRoot = process.cwd();
+
+    const ptau = config.ptau ? path.join(projectRoot, config.ptau) : path.join(os.tmpdir(), ".zkit/.ptau");
```

Comment:
Wouldn't it be better to store the .ptau files in the hostdir?

Since the tmpdir is usually cleared after a system reboot